mautic/mautic
# Removing production assets and adding step to composer.json

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [n]
| New feature/enhancement? (use the a.x branch)      | [y]
| Deprecations?                          | [n]
| BC breaks? (use the c.x branch)        | [n]
| Automated tests included?              | [n] 
| Issue(s) addressed                     | 

#### Description:

The fix above makes sure that https://github.com/mautic/mautic/pull/10647 is no longer needed everytime a release is made. We can also remove the generated assets from the github repo as the tarball that one can download has the newest assets and using composer install it will also generate the assets.

This is similar to https://github.com/mautic/mautic/pull/9736. 

#### Steps to test this PR:

1. run composer install
2. check if the app.css, app.js, libraries.css, libraries.js & mautic-form.js file are being generated.
3. check if you can install mautic and if it works as expected



<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10655"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
